client/blocks/login/index.jsx

```diff
@@ -3,148 +3,56 @@
  */
```

Comment:
Ignore the diff on this file, just read the code as a whole. Its previous content was basically moved to `client/blocks/login/login-form.jsx`